tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -70,6 +70,16 @@ steps:
       --retry $maximumRetryIfNetworkError \
       --verbose
 
+      if [[ "$(release)" == "release" && ${{ parameters.feedKind }} == "internal-build" ]]; then
+        flub promote package \
+        --version "$(SetVersion.version)" \
+        --orderFile $orderFile \
+        --token ":$ADO_API_TOKEN" \
+        --verbose
+      fi
+
       # Clean up; remove the npmrc
       rm ~/.npmrc
       exit 0
+  env:
+    ADO_API_TOKEN: $(System.AccessToken)
```

Comment:
Move this to be above the script property. Generally the script is the last property since the other properties influence it, and it's easier to see all of that up front rather than having to read the script _then_ notice there's an env variable set.